uk.co.compucorp.civicrm.tasksassignments/js/src/crm-tasks-workflows/controllers/casetype.extended.controller.js

```diff
@@ -0,0 +1,45 @@
+/* global angular */
+
+(function (angular) {
+  'use strict';
+
+  angular.module('crm-tasks-workflows.controllers')
+    .controller('CaseTypeExtendedController', CaseTypeExtendedController);
+
+  CaseTypeExtendedController.$inject = [
+    '$log', '$scope', 'crmApi'
+  ];
+
+  function CaseTypeExtendedController ($log, $scope, crmApi) {
+    $log.debug('Controller: CaseTypeExtendedController');
+
+    $scope.activityTypeOptions = [];
+
+    (function init () {
+      fetchActivityTypes();
+    })();
+
+    /**
+     * Fetch activity types which are components of CiviTask or CiviDocument
+     *
+     * @return {Promise}
+     */
+    function fetchActivityTypes () {
+      return crmApi('OptionValue', 'get', {
```

Comment:
```js
vm.loading.options = true;

crmApi(...)
  .then(...)
  .finally(function () {
    vm.loading.options = false;
  });
```

Reply:
Oops, I forgot to add this, apologies. Done.